src/components/button.js

```diff
@@ -0,0 +1,43 @@
+import React, { useRef, useEffect } from 'react'
+import { Sampler } from 'tone'
+
+const Button = (props) => {
+
+  useEffect(() => {
+    document.addEventListener('keydown', handleKeydown)
+    window.focus()
+  }, [])
+
+  const { text, audio } = props;
+  const sound = useRef(null);
+
+  const handleKeydown = (e) => {
+    if(e.keyCode === text.charCodeAt()) {
+      sound.current.play();
```

Comment:
im not sure where the issue is with your buttons not playing but another way to do it would be e.key ==="q"